eb_hooks.py

```diff
@@ -251,11 +251,17 @@ def pre_configure_hook_wrf_aarch64(self, *args, **kwargs):
     - patch arch/configure_new.defaults so building WRF with foss toolchain works on aarch64
     """
     if self.name == 'WRF':
+        wrfversion = int((self.version).replace(".",""))
         if get_cpu_architecture() == AARCH64:
             pattern = "Linux x86_64 ppc64le, gfortran"
             repl = "Linux x86_64 aarch64 ppc64le, gfortran"
-            self.cfg.update('preconfigopts', "sed -i 's/%s/%s/g' arch/configure_new.defaults && " % (pattern, repl))
-            print_msg("Using custom preconfigopts for %s: %s", self.name, self.cfg['preconfigopts'])
+            if wrfversion <= 39:
```

Comment:
@TopRichard It's better to do version checks with `LooseVersion`, and not convert to an integer (consider WRF versions like `3.9.1.1`):

```suggestion
            if LooseVersion(self.version) <= LooseVersion('3.9.0'):